pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -362,12 +362,17 @@ def channel_contributions_forward_pass(
 
     @property
     def _serializable_model_config(self) -> Dict[str, Any]:
+        def ndarray_to_list(d: Dict) -> Dict:
+            new_d = d.copy()  # Copy the dictionary to avoid mutating the original one
+            for key, value in new_d.items():
+                if isinstance(value, np.ndarray):
+                    new_d[key] = value.tolist()
+                elif isinstance(value, dict):
+                    new_d[key] = ndarray_to_list(value)
+            return new_d
```

Comment:
To be future-proof I would make this function recursive when dict values are dicts themselvels. 

In some hierarchical models we did for client projects we had to define priors as nested dicts.